src/bench/wallet_migration.cpp

```diff
@@ -42,6 +42,11 @@ static void WalletMigration(benchmark::Bench& bench)
         const CScript& script = scripts_watch_only.emplace_back(GetScriptForDestination(dest));
         assert(legacy_spkm->LoadWatchOnly(script));
         assert(wallet->SetAddressBook(dest, strprintf("watch_%d", w), /*purpose=*/std::nullopt));
+        {
+            WalletBatch batch(wallet->GetDatabase());
```

Comment:
Can create a single batch instance outside the loop and use it everywhere. The "database" on this test is a plain map in memory. No thread synchronization is needed, writes are done sequential.